VimeoUpload/Upload/Operations/Async/ExportSessionExportCreateVideoOperation.swift

```diff
@@ -162,9 +162,23 @@ open class ExportSessionExportCreateVideoOperation: ConcurrentOperation
                     return
                 }
                 
-                if let error = operation.error
+                if var error = operation.error
                 {
-                    strongSelf.error = error
+                    var userInfo = error.userInfo
+                    
+                    
+                    let asset = AVURLAsset(url: url)
+                    do
+                    {
+                        let fileSize = try asset.fileSize()
+                        userInfo.append([UploadErrorKey.FileSize.rawValue : fileSize])
+                    }
+                    catch
+                    {
+                        
```

Comment:
If using do-catch, we should handle the error here if calculating the file size fails. If there's no need to handle the error (e.g. we just don't append file size) can we then do 

```swift
if let fileSize = try? asset.fileSize()
{
     userInfo.append([UploadErrorKey.FileSize.rawValue : fileSize])
} 
```